cuesubmit/cuesubmit/ui/SettingsWidgets.py

```diff
@@ -54,9 +54,13 @@ class InMayaSettings(BaseSettingsWidget):
     def __init__(self, cameras=None, filename=None, parent=None, *args, **kwargs):
         super(InMayaSettings, self).__init__(parent=parent)
         self.mayaFileInput = Widgets.CueLabelLineEdit('Maya File:', filename)
+        self.fileFilters = ['Maya Ascii file (*.ma)',
```

Comment:
I think the filter options should be moved to the constants file and in the future to a config files (On a future PR, unrelated to this). 

https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/cuesubmit/cuesubmit/Constants.py

Reply:
Sure, I added those constants for Maya, Nuke and Blender file filters.